### DISCUSSION TITLE
Offer to help and triage

### DISCUSSION THREAD

**Discussion by Shinigami92** [faker-js/faker#37] [General]
Hello @damienwebdev, @clarkerican @mmahalwy and @griest024

With this post I am willing to help with moderating (and possible also later contribute code) this repo.

### Background

Since March 2021 I'm member of the Vite Team (https://github.com/vitejs/vite/discussions/2601#discussioncomment-503779)

I'm very active in Discord and @damienwebdev already provided me a `@moderator` role. :slightly_smiling_face: 
Also I like to help to grow a community that pay attention to inclusiveness.

### What would I like to do?

I would like to create [**labels**](https://github.com/vitejs/vite/labels) and help [**review PRs**](https://github.com/vitejs/vite/pulls?q=is%3Apr+reviewed-by%3Ashinigami92+) with the ability to provide a green tick approval.

### What I wont like to do!

Direct access to **main** branch. Everything should be done via a PR and you will have the last word.

---

Please tell me if I need to do anything to earn your trust first :slightly_smiling_face:

**Comment by damienwebdev** [comment]
I've created a @faker-js/faker-js-triage team that has Triage access and added you @Shinigami92
